smarts/sstudio/genhistories.py

```diff
@@ -803,6 +803,10 @@ def _check_args(args) -> bool:
         with open(args.dataset, "r") as yf:
             dataset_spec = yaml.safe_load(yf)["trajectory_dataset"]
 
+    if dataset_spec.get("input_path") == "<PATH TO FILE GOES HERE>":
```

Comment:
I would suggest a default here as well.
```suggestion
    default_input_path = "<PATH TO FILE GOES HERE>"
    if dataset_spec.get("input_path", default_input_path) == default_input_path:
```